packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -25,6 +26,17 @@ The following Role types are provided and supported:
 | `SUPER_ADMIN`                           | Super Administrator                 |                                       |
 | `USER_ADMIN`                            | Group Administrator                 | [Groups](/docs/reference/api/groups/) |
 
+### IAM-based Standard Role Types
+<ApiLifecycle access="ea" />
+We also support the following IAM-based standard roles:
+| Role type                                               | Label                            | Permissions                                   |
+| :------------------------------------------------------ | :------------------------------- | :-------------------------------------------- |
+| `IGA_CERTIFICATIONS_ADMIN` <ApiLifecycle access="ea" /> | OKTA Access Certifications Admin | `okta.governance.accessCertifications.manage` |
```

Comment:
We had such a discussion on this topic before.

So, we have two versions of label for standard roles:
- Label in built-in version. This is the value for the `name` column in `CustomeRoles` table. It has to be unique. It's not the text shown in UI which is the label in localized version. Adding an OKTA prefix to build-in version is to avoid naming conflict with customer-created role.
- Label in localized version. The label showing up in UI will be translated into localized text. Even for english language locale, the label in built-in version is NOT used directly as the localized label. I assume the localized label has no OKTA prefix and if it does then we need a bug jira. 

@alifatolahi-okta, I feel like the label in built-in version is more for internal use. It sounds like it's not a good idea to expose that to the end user. thoughts?

Reply:
There is no localization for standard role names right now, for existing OOTB roles also we simply surface the DB label in public API. If these names have to change, and it's your call really, then it will have to change in code too.

Reply:
+1 for removing "OKTA" label...found it jarring - is this a problem for our release timelines @alifatolahi-okta / @rogertang-okta or it could be addressed as bug in July?

Reply:
I created a JIRA for that. Please let me know we want to proceed it. 

https://oktainc.atlassian.net/browse/OKTA-509517

Reply:
@rogertang-okta I think that we should proceed with this change. I understand the risk of collision if a customer already has created a custom admin role with this name. However, seeing as none of these capabilities are included in the IAM Framework today, I think that this risk is very low.